weed/s3api/s3api_object_multipart_handlers.go

```diff
@@ -205,7 +206,13 @@ func (s3a *S3ApiServer) PutObjectPartHandler(w http.ResponseWriter, r *http.Requ
 	bucket, object := xhttp.GetBucketAndObject(r)
 
 	uploadID := r.URL.Query().Get("uploadId")
-	err := s3a.chkUploadID(bucket, object, uploadID)
+	exists, err := s3a.exists(s3a.genUploadsFolder(bucket), uploadID, true)
```

Comment:
this seems unnecessary. If the request can have the correct uploadId, it is unlikely the uploadId does not exist.